webapp/src/apis/rsu-api.ts

```diff
@@ -129,6 +130,15 @@ class RsuApi {
       tag: 'wzdx',
     })
 
+  // Moove AI
+  postMoveAiData = async (token: string, body: Object, url_ext: string = ''): Promise<ApiMsgRespWithCodes<any>> =>
```

Comment:
suggestion (non-blocking) - Add type def for the returned data

Reply:
Addressed